translation_helper/translate.py

```diff
@@ -153,17 +154,17 @@ def choose_by_language(input_json, chosen_language):
                         actions += " %s " % phrase_translations["or"]
                     actions += ("<r>%s</r>" if action["kwargs"]["side_effect_name"] == "STRESS" else "%s") % \
                                actions_by_id[action["kwargs"]["pk"]]
-                card_translation["ability_text"] = "<i>" + phrase_translations["requires"] % actions + "</i>" + "%LINEBREAK%" + \
-                                          card_translation["ability_text"]
+                card_translation["ability_text"] = "<i>" + phrase_translations[
+                    "requires"] % actions + "</i>" + "%LINEBREAK%" + card_translation["ability_text"]
             # if restricted to given base size, add a note to the text
             elif restriction[0]["type"] == "SHIP_SIZE":
                 sizes = ""
                 for size in restriction:
                     if sizes:
                         sizes += " %s " % phrase_translations["or"]
                     sizes += phrase_translations[size["kwargs"]["ship_size_name"]]
-                card_translation["ability_text"] = "<i>" + phrase_translations["only"] % sizes + "</i>" + "%LINEBREAK%" + \
-                                          card_translation["ability_text"]
```

Comment:
Is what my IDE done not PEP 8 valid? It looks more readable not to break inside the [ ] But hey, I really don't care ;-)